README.md

```diff
@@ -2,16 +2,18 @@
 
 This is a demonstration of AR running in Ionic 4 for mobile devices.
 
-## Running on an android device
+## Running locally
 
-1. Install the packages
+`$ ionic serve`
 
-`$ npm install`
+## Running in an android simulator
```

Comment:
So accessing camera works on simulator and AR model comes up?

Reply:
Yes, although it requires a bit of extra setup to [get the android emulator to use the webcam](https://stackoverflow.com/a/30792615).

However, it is still better and easier to test it from a real device.